docs/Concepts/Consensus-Protocols/Overview-Consensus.md

```diff
@@ -6,10 +6,17 @@ description: Besu consensus protocols
 
 Besu implements the following consensus protocols:
 
-* Ethash (proof of work)
-* [Clique](../../HowTo/Configure/Consensus-Protocols/Clique.md) (proof of authority)
-* [IBFT 2.0](../../HowTo/Configure/Consensus-Protocols/IBFT.md) (proof of authority)
-* [QBFT](../../HowTo/Configure/Consensus-Protocols/QBFT.md) (proof of authority).
+* [QBFT](../../HowTo/Configure/Consensus-Protocols/QBFT.md) (proof of authority): QBFT is the recommended
```

Comment:
We don't need to reuse the consensus protocol name in these descriptions.

"QBFT: the recommended..."